src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        String[][] che = new String[records.length][1];
```

Comment:
that's overcomplicated, complete task with single loop and you don't need 2d arr